libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/instruction/InstructionStepResources.java

```diff
@@ -98,7 +98,7 @@ private void thenStep(LegStep upcomingStep, LegStep followOnStep, double current
     thenStepManeuverType = followOnStep.maneuver().type();
     thenStepManeuverModifier = followOnStep.maneuver().modifier();
     // Should show then step if the upcoming step is less than 25 seconds
-    shouldShowThenStep = upcomingStep.duration() <= (25d * 1.2d) && currentDurationRemaining <= (70d * 1.2d);
+    shouldShowThenStep = upcomingStep.duration() <= (25d * 1.2d) && currentDurationRemaining <= 70d;
```

Comment:
Could we fix the variable name to reflect what actually is doing? This way comment will become superfluous so it won't be necessary.

Reply:
What about [decomposing conditional](https://refactoring.com/catalog/decomposeConditional.html) extracting that complicated expression into a well-named `private` method? Conditionals tend to get more and more complicated in their logic over time.

Reply:
Magic numbers.